aiogram/types/update.py

```diff
@@ -15,6 +16,13 @@
     from .pre_checkout_query import PreCheckoutQuery
     from .shipping_query import ShippingQuery
 
+T = TypeVar("T")
+
+
+def lru_cache(func: Callable[..., T]) -> T:
```

Comment:
This hack can be used in other places later, so that's good reason to move this function to utils